### PR TITLE
[New Rule] Multiple Alerts in Different ATT&CK Tactics on a Single Host

### LINKED ISSUE #1597: [New Rule] Threshold rule to detect multiple different Mitre Tactics on the same host in the last 24h
## Description

A Threshold rule that looks for unique count of more than 2 different `kibana.alert.rule.threat.tactic.name` values for a single host.name in the last 24h and generates a critical alert when they are observed. This could be an indicator of an ongoing attack impacting multiple parts of the kill chain.

This is an example rule I created on my cluster
Screen Shot 2021-10-29 at 14 53 37

### Target indexes

`.siem-signals-*`

### Additional requirements

Detection rules need to be configured with `kibana.alert.rule.threat.tactic.name` as much as possible.

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

Resolves #1597 

## Summary

This rule uses alert data to determine when multiple alerts in different phases of an attack involving the same host are
triggered. Analysts can use this to prioritize triage and response, as these hosts are more likely to be compromised.

### CODE REVIEW

**Review by Samirbous — APPROVED** [review]

File: rules/cross-platform/multiple_alerts_different_tactics_host.toml
Comment (brokensound77) [inline rules/cross-platform/multiple_alerts_different_tactics_host.toml:36]: nit

```suggestion
signal.rule.name:* and kibana.alert.rule.threat.tactic.id:*
```

**Review by brokensound77 — APPROVED** [review]
LGTM but we should wait until the timeline template is merged

**Pull request merged**
